.github/workflows/detekt-with-type-resolution.yaml

```
@@ -9,6 +9,42 @@ on:
       - '*'
 
 jobs:
+  without-type-resolution:
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.head_commit.message, 'ci skip') }}
+    env:
+      GRADLE_OPTS: -Dorg.gradle.daemon=false
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Cache Gradle Folders
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/
+            ~/.gradle/wrapper/
+          key: cache-gradle-${{ hashFiles('detekt-bom/build.gradle.kts') }}
+          restore-keys: |
+            cache-gradle-
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Build detekt and skip checks
+        run: ./gradlew build moveJarForIntegrationTest shadowJarExecutable -x check
+
+      - name: Run detekt-cli with argsfile, override the exit code
+        run: java -jar ./detekt-cli/build/run/detekt "@./config/detekt/argsfile" || true
```

You are using `shadowJarExecutable`. So you could run this like 
```suggestion
        run: ./detekt-cli/build/run/detekt "@./config/detekt/argsfile" || true
```

---

Good suggestion. I will pick it up in my own commit bundled with the previous comment.